src/test/resources/features/refapp-2.x/015-reports/reports.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Reports Management
+
+  Background:
+    Given a user go to system administartion app
+    Then the system loads system administrationpage
+     
+  @selenium
+  @report
+  Scenario: Report Management
+    When user clicks on advanced administration page
+    And user clicks on report administration link
+    Then the system loads manage reports page
+    And user clicks on run button from manage Reports page
+    And user enters start date
+    Then user enters end date
```

Comment:
isn't this gherkin syntax supposed to be `And use enter end date`?

Reply:
Nope thanks that scenario isnt well readable, it should be **user** not **use**

Reply:
I meant the gherkin syntax of `Then` Vs `And`